include/tvm/relay/transform.h

```diff
@@ -47,6 +47,18 @@ using PassInfoNode = tvm::transform::PassInfoNode;
 using PassContext = tvm::transform::PassContext;
 using PassContextNode = tvm::transform::PassContextNode;
 using Sequential = tvm::transform::Sequential;
+using FTVMRelayToTIR = tvm::transform::Pass;
```

Comment:
Looks like this one has lost its documentation during the refactor...

Reply:
added, thanks!